src/providers/TransactionBuilderContextProvider.tsx

```diff
@@ -619,6 +619,7 @@ const TransactionBuilderContextProvider = ({
         result?.errorMessage ||
         (!result?.transactionHash?.length)
       ) {
+        showAlertModal(result.errorMessage ?? 'Unable to send Polygon transaction!');
```

Comment:
Assist the user in what to do next. For example, should they try again?

Reply:
That's an edge case where I am not sure what to do if the transaction isn't enough to pay for gas on destination chain or the quote from lifi doesn't go through